omod/src/main/resources/META-INF/web-fragment.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wrote this file and am hoping to find a way to get this file to actually be integrated into OpenMRS' web.xml file without modifying openmrs-core. See related https://www.ibm.com/support/knowledgecenter/SS7K4U_9.0.0/com.ibm.websphere.zseries.doc/ae/cweb_webfragments.html and https://www.oracle.com/technetwork/articles/javaee/javaee6overview-part2-136353.html.

This file is related to the issue described in `4.` in the Readme.

Unfortunately, I have been unable to get Tomcat/Jetty to respect this web fragment config.

Reply:
Is this of help? https://wiki.openmrs.org/display/docs/Module+Servlets